src/Goerli-DssSpellCollateral.sol

```diff
@@ -0,0 +1,192 @@
+
+    // --- Rates ---
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
+    //
+
+    // --- Math ---
+
+    function onboardNewCollaterals() internal {
+        // ----------------------------- Collateral onboarding -----------------------------
+        //  Add ______________ as a new Vault Type
+        //  Poll Link:
+
+        // DssExecLib.addNewCollateral(
+        //     CollateralOpts({
+        //         ilk:                   ,
+        //         gem:                   ,
+        //         join:                  ,
+        //         clip:                  ,
+        //         calc:                  ,
+        //         pip:                   ,
+        //         isLiquidatable:        ,
+        //         isOSM:                 ,
+        //         whitelistOSM:          ,
+        //         ilkDebtCeiling:        ,
+        //         minVaultAmount:        ,
+        //         maxLiquidationAmount:  ,
+        //         liquidationPenalty:    ,
+        //         ilkStabilityFee:       ,
+        //         startingPriceFactor:   ,
+        //         breakerTolerance:      ,
+        //         auctionDuration:       ,
+        //         permittedDrop:         ,
+        //         liquidationRatio:      ,
+        //         kprFlatReward:         ,
+        //         kprPctReward:
+        //     })
+        // );
+
+        // DssExecLib.setStairstepExponentialDecrease(
+        //     CALC_ADDR,
+        //     DURATION,
+        //     PCT_BPS
+        // );
+
+        // DssExecLib.setIlkAutoLineParameters(
+        //     ILK,
+        //     AMOUNT,
+        //     GAP,
+        //     TTL
+        // );
+
+        // ChainLog Updates
+        // Add the new flip and join to the Chainlog
+        // address constant CHAINLOG        = DssExecLib.LOG();
+        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
+        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
+        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    }
+
+    function offboardCollaterals() internal {
+        // Offboard UNI-A
+        // https://vote.makerdao.com/polling/QmSfLS6V#poll-detail
+        // https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+
+        DssExecLib.setIlkLiquidationPenalty("UNI-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("UNI-A", 0);
+        DssExecLib.linearInterpolation({
+            _name:      "UNI-A Offboarding",
+            _target:    DssExecLib.spotter(),
```

Comment:
Neat - I know this was done like this in the previous offboardings, but no need to read from the changelog each time, can store this value locally and use in all the places.